sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4338,6 +4338,7 @@ _ZNK4sycl3_V16device8get_infoINS0_4info6device32atomic_memory_order_capabilities
 _ZNK4sycl3_V16device8get_infoINS0_4info6device32atomic_memory_scope_capabilitiesEEENS0_6detail19is_device_info_descIT_E11return_typeEv
 _ZNK4sycl3_V16device8get_infoINS0_4info6device33ext_intel_gpu_subslices_per_sliceEEENS0_6detail19is_device_info_descIT_E11return_typeEv
 _ZNK4sycl3_V16device8get_infoINS0_4info6device33ext_oneapi_max_global_work_groupsEEENS0_6detail19is_device_info_descIT_E11return_typeEv
+# TODO: Remove below symbol in next ABI break 
```

Comment:
These will fail when we remove the deprecated aspects in sycl/include/sycl/info/aspects_deprecated.def, right? If so, these comments aren't needed here.